plugins/inputs/nats_consumer/nats_consumer.go

```diff
@@ -106,6 +107,14 @@ func (n *natsConsumer) Start(acc telegraf.Accumulator) error {
 		options = append(options, nats.UserCredentials(n.Credentials))
 	}
 
+	if n.NkeySeed != "" {
```

Comment:
It would be good to verify that the file exists and is readable in `Init()` to save users some possible confusion.

Reply:
Currently, there are some descriptive error messages are returned from NATS.

If the file doesn't exist the error message is:
```sh
E! [telegraf] Error running agent: starting input inputs.nats_consumer::test-nats-consumer: nats: open etc/telegraf/seed.txt: no such file or directory
```

And if it's unreadable:
```sh
E! [telegraf] Error running agent: starting input inputs.nats_consumer::test-nats-consumer: nats: open etc/telegraf/seed.txt: permission denied
```

Is this sufficient or is there something else I should do?

Reply:
That's fine! Thanks for confirming that.